nav2_simple_commander/nav2_simple_commander/costmap_2d.py

```diff
@@ -33,8 +34,7 @@ def __init__(self, occupancy_map):
         self.global_frame_id = occupancy_map.header.frame_id
         self.costmap_timestamp = occupancy_map.header.stamp
         # Extract costmap
-        self.costmap = np.array(occupancy_map.data, dtype=np.int8).reshape(
-            self.size_y, self.size_x)
+        self.costmap = np.array(occupancy_map.data, dtype=np.int8)
```

Comment:
should be unsigned?

Reply:
I refer to the cosmap_2d cpp code. 
[costmap_2d.hpp](https://github.com/ros-planning/navigation2/blob/main/nav2_costmap_2d/include/nav2_costmap_2d/costmap_2d.hpp)
The return value of costmap is unsigned char.
I think the python should also use the uint type.
What do you think?

Reply:
Agreed

Reply:
Okay I will change and notify again 😄